### DISCUSSION TITLE
Changing powder averaging methods

### DISCUSSION THREAD

**Discussion by mjunior-NMR** [deepanshs/mrsimulator#305] [Q&A]
Dear MRSimulator developers and community,
I would like to have more information about which method of powder averaging MRSimulator is using to create powder patterns. Also, I want to know if (and how) it would be possible for the user to change these methods for specific simulations.
Thank you very much,
Marcos Oliveira.

**Comment by deepanshs** [comment]
Hi Marcos,

Mrsimulator uses Alderman and Grant's approach of octahedron sampling and interpolation ([https://doi.org/10.1063/1.450211](https://doi.org/10.1063/1.450211)). The two parameters that generate this sampling pattern are  `integration_volume` and `integration density`. These parameters are described in the [doc](https://mrsimulator.readthedocs.io/en/stable/user_guide/simulator/simulator.html#integration-density).

In the current version, it is not possible to change the powder averaging method, but it is possible to include it in a future version. You can submit a feature request in the [feature request](https://github.com/deepanshs/mrsimulator/issues/new?assignees=&labels=type%3AEnhancement&projects=&template=FEATURE_REQUEST.md&title=) section.

Best,
Deepansh

**Comment by mjunior-NMR** [comment]
Dear Deepansh,
thank you very much for your attention. It was really helpful.

If you could answer one other question I would appreciate a lot. Do you
know how the eps parameter in the extended Czjzek model in MRSilmulator
relates to the sigma parameter in ssnake?
Thank you very much,
Marcos.
--

Prof. Dr. Marcos de Oliveira Junior
Grupo de Ressonância Magnética, Espectroscopia e Magnetismo
Instituto de Física de São Carlos, Universidade de São Paulo (IFSC/USP)
Av. Trabalhador Sãocarlense 400, 13566-590, São Carlos - SP, Brasil
Sala 19 - Bloco F2 - 1º andar



Em qua., 29 de nov. de 2023 às 13:03, Deepansh Srivastava <
***@***.***> escreveu:

**Reply by deepanshs** [reply]
Dear Marcos,

We have documented the parameter `eps` in the [docs](https://mrsimulator.readthedocs.io/en/latest/theory/models.html#extended-czjzek-distribution). The original definition is from [Caer et. al.](https://doi.org/10.1088/0953-8984/22/6/065402), Eq. 10. According to this definition, $\rho$ (which is parametrically dependent on $\epsilon$ `eps`) is the standard deviation of the multivariate Gaussian, the same as the $\sigma$ parameter from ssnake.

Please note that this assessment is based on the definition of standard deviations from [Meerten et. al.](https://www.sciencedirect.com/science/article/pii/S1090780719300308#b0065) and [Caer et. al.](https://doi.org/10.1088/0953-8984/22/6/065402). I haven't compared and verified it with simulation. If you do verify simulations, please share your findings with us.

Thank you,
Deepansh

**Comment by mjunior-NMR** [comment]
Dear Deepansh,
I am currently in a conference. As soon as I get back to the office, I send
you an example.
Best regards,
Marcos.
--

Prof. Dr. Marcos de Oliveira Junior
Grupo de Ressonância Magnética, Espectroscopia e Magnetismo
Instituto de Física de São Carlos, Universidade de São Paulo (IFSC/USP)
Av. Trabalhador Sãocarlense 400, 13566-590, São Carlos - SP, Brasil
Sala 19 - Bloco F2 - 1º andar



Em seg., 4 de dez. de 2023 às 15:27, Deepansh Srivastava <
***@***.***> escreveu:
